### PR TITLE
[Merged by Bors] - chore: Remove `Cat` suffixes

### PR DESCRIPTION
**Pull request by YaelDillies**
These names needn't change in the first place.

---

I am not reverting the `Mon → MonCat` rename here just in case people want to do funky stuff with `Mon_`.
[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Category/NonemptyFinLinOrd.lean [hunk 1]
```diff
@@ -53,51 +53,51 @@ instance (α : Type _) [NonemptyFinLinOrd α] : NonemptyFinLinOrd αᵒᵈ :=
   { OrderDual.fintype α with }
 
 /-- The category of nonempty finite linear orders. -/
-def NonemptyFinLinOrdCat :=
+def NonemptyFinLinOrd :=
   Bundled NonemptyFinLinOrd
```
Comment (Parcly-Taxel) [inline Mathlib/Order/Category/NonemptyFinLinOrd.lean:57]: I think this is the reason the whole scheme is not a [useless use of `cat`](https://en.wikipedia.org/wiki/UUOC); the suffix helps with disambiguating a few things.

**Comment by joelriou** [comment]
The `Cat` suffix was added intentionally, see 
https://github.com/leanprover-community/mathlib4/wiki/Porting-wiki#categories "Categories are suffixed with `Cat`"

File: Mathlib/Order/Category/NonemptyFinLinOrd.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Order/Category/NonemptyFinLinOrd.lean:57]: You'll notice that `NonemptyFinLinOrd` is an auxiliary definition in the construction of `NonemptyFinLinOrdCat`. I'll rename it.

**Comment by YaelDillies** [comment]
Yes, I know, but this addition is a last resort when the name of the `foo` clashes with the name of the category `Foo` of bundled `foo`s. And I've specifically renamed all the order categories so that the names wouldn't clash.

**Comment by semorrison** [comment]
I propose we continue this as https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Naming.20convention/near/354574169.

**Review by Parcly-Taxel — APPROVED** [review]

**Comment by semorrison** [comment]
Consensus is that we're not doing this, so I'll close.

**Comment by jcommelin** [comment]
What's the status here? Should we have more discussion on zulip?

**Comment by YaelDillies** [comment]
I think most people don't really care either way, and Joël is slightly in favour?

**Comment by semorrison** [comment]
I'm in favour.

bors d+

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**